components/list/list-item-link-mixin.js

```diff
@@ -29,7 +29,10 @@ export const ListItemLinkMixin = superclass => class extends ListItemMixin(super
 				outline: none;
 				width: 100%;
 			}
-		` ];
+			:host([skeleton]) a[href] {
```

Comment:
The skeletons were not covering the link in w2d. I didn't dig into why. We had originally been holding off adding `actionHref` until the skeleton was gone but it was making things more complicated than necessary.

Reply:
Ah k.  I'd like to see if I can reproduce this in the demo page. This change might be what we want but I'd like to understand the root cause first.

Reply:
Ok, I think I see what's happening here.  The list-item layout stacks its action `<a>` on top of the content so that when the user clicks on the item it navigates, or if they right-click then they get all the standard browser actions for links etc. This `<a>` is typically invisible and fills the same dimensions as the content but the user can still interact with it while the skeleton is active.  So... hiding this element I think is the right thing here.